index.html

```diff
@@ -6,19 +7,42 @@
       <meta http-equiv="X-UA-Compatible" content="IE=edge">
       <meta name="description" content="">
       <meta name="viewport" content="width=device-width, initial-scale=1">
-      <title>Sorting the slow way</title>
-      <link rel='stylesheet' type='text/css' href='//css/main.css'/>
-      <script type='text/javascript' src='//js/main.js'></script>
+      <title>Sorting the Slow Way</title>
+      <link href="css/meyersreset.css" type="text/css" rel="stylesheet"/>
+      <link href="css/960_16_col.css" type="text/css" rel="stylesheet"/>
+      <link href="css/main.css" type="text/css" rel="stylesheet"/>
+      <link href='http://fonts.googleapis.com/css?family=Audiowide' rel='stylesheet' type='text/css'>
     </head>
-    <body>
-      <h1>Bogo Sort<h1>
-      <h2>Sorting that rocks</h2>
-      <a>Bogo sort....blah blah blah</a>
-      <button>Go!</button>
-      <p></p>
-      <table>
-
+    <body class="container_16">
+      <h1 class="grid_16 container_16 title ">Bogosort</h1>
+      <h2 class="grid_16 container_16 subtitle">Sorting that Rocks</h2>
+      <img class="imageholder" src="img/3.jpg"/>
+      <article class="grid_16 container_16 mainbody"/>
+        <p>Bogosort is a sorting algorithm.  It is also known by other names, including monkey sort, and shotgun sort.  It is not the fastest, in fact it can probably considered the slowest becuase but it rocks as an algorithm. Everytime you try the Bogosort your elements may potentialy never sort.</p>
+        <p>The algorithm for this sort starts with an array of elements. Check the array to see if it is sorted or not.  If not, randomize the entire array and check again.
+        </p>
+        <p>*****Bewarned that arrays greater than 6 or 7 may take a very long time*****</p>
+        <p></p>
+        <section>
+          <input type=radio name="speedBut" id="slowSpeed">slow</input>
+          <input type=radio name="speedBut" id="mediumSpeed" checked="true">medium</input>
+          <input type=radio name="speedBut" id="fastSpeed">fast</inpput>
+        <section>
+      </article>
+      <button class="clickme playbutton">Lets Sort!</button>
+      <ul class="sortlist outputcontainter grid_8 container_8">
```

Comment:
For bonus style points, use kebab-case: `sort-list` and `click-me` and `play-button`, etc...